### PR TITLE
Document how to expose collector out of the cluster

### LINKED ISSUE #55: Move collector route to a separate file + test it

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,18 @@ appropriate to create a `PersistentVolumeClaim`/`PersistentVolume` and use it in
...
+Collector is by default accessible only to services running inside the cluster. 
+The easiest is to expose HTTP `jaeger-collector-http` port via OpenShift Route:
+
+```bash
+oc create route edge --service=jaeger-collector --port jaeger-collector-http --insecure-policy=Allow
+```
+
+This allows to send data from clients directly to Collector via HTTP senders. If you want to use Agent then use 
+[ExternalIP or NodePort](https://docs.openshift.com/container-platform/3.3/dev_guide/getting_traffic_into_cluster.htm)
+to expose Collector service.
```
Comment (jpkrohling) [inline README.md]: Can you leave a comment about the security aspect, like we discussed yesterday? Something like:

> Note that doing so will open the collector to be used by any external party, who will then be able to create arbitrary spans. It's advisable to put an OAuth Security Proxy in front of the collector and expose this proxy instead.

**Comment by pavolloffay** [comment]
Or we can define this in a separate file which could be tested

File: README.md [hunk 2]
```diff
@@ -111,6 +111,22 @@ appropriate to create a `PersistentVolumeClaim`/`PersistentVolume` and use it in
 Cassandra deployment does not support deleting pods or scaling down, as this might require
 administrative tasks that are dependent on the final deployment architecture.
 
+## Exposing Jaeger-Collector out of Cluster
```
Comment (objectiser) [inline README.md]: "out" -> "outside"

File: README.md [hunk 3]
```diff
@@ -111,6 +111,22 @@ appropriate to create a `PersistentVolumeClaim`/`PersistentVolume` and use it in
...
+Collector is by default accessible only to services running inside the cluster. 
+The easiest is to expose HTTP `jaeger-collector-http` port via OpenShift Route:
```
Comment (objectiser) [inline README.md]: Maybe reword: "The easiest approach to expose the collector outside of the cluster is via the `jaeger-collector-http` HTTP port using an OpenShift Route:"

File: README.md [hunk 4]
```diff
@@ -111,6 +111,22 @@ appropriate to create a `PersistentVolumeClaim`/`PersistentVolume` and use it in
...
+
+```bash
+oc create route edge --service=jaeger-collector --port jaeger-collector-http --insecure-policy=Allow
+```
+
+This allows to send data from clients directly to Collector via HTTP senders. If you want to use Agent then use
```
Comment (objectiser) [inline README.md]: "This allows clients to send data directly to the Collector ..."

Comment (objectiser) [reply README.md]: "use the Agent then ..."

File: README.md (hunk 1 above)
Comment (objectiser) [reply README.md]: nit: "expose the Collector service"

**Comment by pavolloffay** [comment]
@kevinearls would it be possible to create a test for collector route?

**Comment by pavolloffay** [comment]
I will merge this, we can reiterate and move route definition to the separate file
